variables.tf

```diff
@@ -4,139 +4,269 @@ variable "name" {
 }
 
 variable "ses_configuration_arn" {
-  type = string
+  description = "SES configuration ARN"
+  type        = string
 }
 
 variable "ses_from_email" {
-  type = string
+  description = "From email address for SES"
+  type        = string
 }
 
 variable "auth_sms_message" {
-  type    = string
-  default = "Este código te lo envía {####}"
+  description = "Authentication SMS message"
+  type        = string
+  default     = "Este código te lo envía {####}"
 }
 
-variable "domain" {
-  type = string
+variable "html_email_message" {
+  description = "Custom HTML email message"
+  type        = string
+  default     = ""
 }
 
-variable "certificate_arn" {
-  type = string
+variable "email_verification_subject" {
+  description = "Email verification subject"
+  type        = string
 }
 
-variable "region" {
-  type = string
+variable "auto_verified_attributes" {
+  description = "List of attributes to be auto-verified in Cognito"
+  type        = list(string)
+  default     = []
+}
+
+
+variable "enable_software_token_mfa" {
+  description = "Boolean to enable/disable software token MFA configuration"
+  type        = bool
+  default     = false
+}
+
+variable "email_sending_account" {
+  description = "Email sending account"
+  type        = string
+  default     = "DEVELOPER"  # Valor por defecto razonable
+}
+
+variable "reply_to_email_address" {
+  description = "Reply-to email address"
+  type        = string
+  default     = "admin@default.com"  # Valor por defecto razonable
+}
+
+variable "sms_external_id" {
+  description = "SMS external ID"
+  type        = string
+  default     = ""  # Valor por defecto vacío si es opcional
+}
+
+variable "enable_sms_sns" {
+  description = "Boolean to enable/disable the creation of the SNS SMS policy"
+  type        = bool
+  default     = false
+}
+
+variable "sms_sns_caller_arn" {
+  description = "SNS caller ARN for Cognito SMS configuration"
+  type        = string
+  default     = null
+}
+
+variable "sms_sns_region" {
+  description = "SMS SNS region"
+  type        = string
+  default     = "us-east-1"  # Valor por defecto razonable
+}
```

Comment:
Este valor sobra, debe estar ligado a la variable global de la región, esta variable es redundante

Reply:
The sms_sns_region variable has been removed as it was redundant. The value is now tied to the global region variable, ensuring consistency and reducing unnecessary configuration.